docs/docs/tutorials/real-world-example-with-react/9-authenticated-api.md

```diff
@@ -37,7 +37,7 @@ export class StoriesController {
     story.title = ctx.request.body.title;
     story.link = ctx.request.body.link;
     // Set the current user as the author of the story.
-    story.author = ctx.user;
+    story.author = ctx.user as User;
```

Comment:
Thank you for pointing this out!

Maybe we should change the line 35 here instead with `async createStory(ctx: Context<User>)` to be consistent with the rest of the tutorial. It would also remove the need of casting.

Reply:
Yeah, that is a much nicer solution. Updated the PR.